validation.go

```diff
@@ -47,3 +47,18 @@ func ValidateBucketName(name string) error {
 
 	return nil
 }
+
+func validNumericString(v string) bool {
```

Comment:
Not sure where this is currently used, but I assume it will be as well relevant later? ;-)

Reply:
Oh, actually I think this is a remnant from a previous experiment. I think it can probably be culled, but maybe it might be useful? Not sure. Would you like me to remove it?